pkg/kube/pod_test.go

```diff
@@ -62,17 +77,80 @@ func (s *PodSuite) TearDownSuite(c *C) {
 }
 
 func (s *PodSuite) TestPod(c *C) {
+	// get controllers's namespace
+	cns, err := GetControllerNamespace()
+	c.Assert(err, IsNil)
+
+	// get controller's SA
+	sa, err := GetControllerServiceAccount(fake.NewSimpleClientset())
+	c.Assert(err, IsNil)
+
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
-	pod, err := CreatePod(context.Background(), s.cli, &PodOptions{
-		Namespace:    s.namespace,
-		GenerateName: "test-",
-		Image:        "kanisterio/kanister-tools:0.28.0",
-		Command:      []string{"sh", "-c", "tail -f /dev/null"},
-	})
-	c.Assert(err, IsNil)
-	c.Assert(WaitForPodReady(ctx, s.cli, s.namespace, pod.Name), IsNil)
-	c.Assert(DeletePod(context.Background(), s.cli, pod), IsNil)
+
+	podOptions := []*PodOptions{
+		{
+			Namespace:    s.namespace,
+			GenerateName: "test-",
+			Image:        "kanisterio/kanister-tools:0.28.0",
```

Comment:
please use constants for this image. We should have it defined somewhere

Reply:
there is one in package function's util.go but 
1. that is not exported and 
2. using that is going to introduce import cycle
so I am planning to make constant in the same source file.